src/features/live_sensor/index.ts

```diff
@@ -2,23 +2,23 @@ import { notificationServiceInstance } from '~core/notificationServiceInstance';
 import { toolbar } from '~core/toolbar';
 import { i18n } from '~core/localization';
 import { store } from '~core/store/store';
-import { SENSOR_CONTROL, SENSOR_CONTROL_NAME } from './constants';
+import { SENSOR_CONTROL_ID } from './constants';
 import { LiveSensor } from './LiveSensor';
 
 export const liveSensorsControl = toolbar.setupControl<{
   liveSensor?: LiveSensor;
 }>({
-  id: SENSOR_CONTROL,
+  id: SENSOR_CONTROL_ID,
   type: 'button',
   typeSettings: {
-    name: SENSOR_CONTROL_NAME,
+    name: i18n.t('toolbar.start_sensor_recording'),
```

Comment:
it will become Stop once recording starts though, like the hint on the line below suggests?

Reply:
The hint doesn't seem to have a visual effect on the button. To implement dynamic text change I would need more time and learn more about the sensors recording functionality.

Discussed it with @alesiahil , decided to set a static "Record Sensors" text, for now.